src/validationinterface.h

```diff
@@ -151,7 +159,7 @@ class CMainSignals {
 private:
     std::unique_ptr<MainSignalsInstance> m_internals;
 
-    friend void ::RegisterValidationInterface(CValidationInterface*);
+    friend void ::RegisterSharedValidationInterface(std::shared_ptr<CValidationInterface>);
     friend void ::UnregisterValidationInterface(CValidationInterface*);
```

Comment:
Why not shared_ptr here?

Reply:
re: https://github.com/bitcoin/bitcoin/pull/18338#discussion_r393265428

> Why not shared_ptr here?

This is an existing function the PR isn't changing (and there's no reason to change it)